index.html

```diff
@@ -12,21 +12,52 @@
     <div id="signin_background">
         <h1>Sign in</h1>
 
-        <form class="info-user">
-            <label for="username">Username:</label>
+    <div id="signin-form">
+        <h1>Connection</h1>
+        <form class="InfoUser">
+            <label for="username">Nom d'utilisateur:</label>
             <br>
-            <input type="text" id="username" name="username" placeholder="Enter Username:"><br><br>
+            <input type="text" id="signin-username" name="signin-username"
+                placeholder="Entrez nom d'utilisateur:"><br><br>
 
-            <label for="password">Password:</label>
+            <label for="signin-password">Mot de passe:</label>
             <br>
-            <input type="password" id="password" name="password" placeholder="Enter password:"><br><br>
+            <input type="password" id="signin-password" name="signin-password"
+                placeholder="Entrez mot de passe:"><br><br>
 
-            <input type="submit" value="Submit" id="btnSubmit">
+            <input type="submit" value="Connection" id="signin-submit">
         </form>
+        <p>Vous n'avez pas de compte? <a href="#" onclick="showSignUpForm()">Inscription</a></p>
+    </div>
+
+    <div id="signup-form" class="signup-form">
+        <h1>Inscription</h1>
+        <form class="InfoUser">
+            <label for="name">Nom:</label><br>
+            <input type="text" id="name" name="name" placeholder="Entrez votre nom:"><br><br>
+
+            <label for="email">Courriel:</label><br>
+            <input type="email" id="email" name="email" placeholder="Entrez votre email:"><br><br>
+
+            <label for="signup-username">Nom d'utilisateur:</label><br>
+            <input type="text" id="signup-username" name="signup-username"
+                placeholder="Entrez nom d'utilisateur:"><br><br>
 
+            <label for="signup-password">Password:</label><br>
+            <input type="password" id="signup-password" name="signup-password"
+                placeholder="Entrez mot de passe:"><br><br>
 
+            <label for="confirm-password">Confirm Password:</label><br>
```

Comment:
Ne pas utiliser de \<br>!
```suggestion
            <label for="confirm-password">Confirm Password:</label>
```